app/models/index.rb

```diff
@@ -10,7 +10,7 @@ class Index < ApplicationRecord
 
   def editable_by?(user)
     return false unless user
-    return true if open_to_anyone?
+    return false if authors_locked?
```

Comment:
This seems like it's going to break a lot of things? I think you need return true unless, not return false if.

Reply:
...I think you're right, yes